packages/ledger-icp/candid/ledger.idl.js

```diff
@@ -28,8 +28,6 @@ export const idlFactory = ({ IDL }) => {
     'token_symbol' : IDL.Opt(IDL.Text),
     'transfer_fee' : IDL.Opt(Tokens),
     'minting_account' : TextAccountIdentifier,
-    'maximum_number_of_accounts' : IDL.Opt(IDL.Nat64),
```

Comment:
Fields were removed in https://github.com/dfinity/ic/pull/3314 but no breaking change.